src/api-doc/api-doc.yaml

```diff
@@ -0,0 +1,2869 @@
+---
+openapi: 3.0.0
+info:
+  title: Elevate Mentoring
+  version: 1.0.0
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  description: >-
+    - The Mentoring Service is a centralized Service to support other services. Apis perform operations related to user entities notification etc 
```

Comment:
@ankitpws please update the valid description

Reply:
remove notification

Reply:
fixed